pkg/apis/nsx.vmware.com/v1alpha1/vpcnetworkconfiguration_types.go

```diff
@@ -19,38 +24,38 @@ const (
 // in a Namespace's VPCNetworkConfiguration, the Namespace will use the value
 // in the default VPCNetworkConfiguration.
 type VPCNetworkConfigurationSpec struct {
-	// PolicyPath of Tier0 or Tier0 VRF gateway.
-	DefaultGatewayPath string `json:"defaultGatewayPath,omitempty"`
-	// Edge cluster path on which the networking elements will be created.
-	EdgeClusterPath string `json:"edgeClusterPath,omitempty"`
 	// NSX-T Project the Namespace associated with.
-	NSXTProject string `json:"nsxtProject,omitempty"`
-	// NSX-T IPv4 Block paths used to allocate external Subnets.
-	// +kubebuilder:validation:MinItems=0
-	// +kubebuilder:validation:MaxItems=5
-	ExternalIPv4Blocks []string `json:"externalIPv4Blocks,omitempty"`
-	// Private IPv4 CIDRs used to allocate Private Subnets.
-	// +kubebuilder:validation:MinItems=0
-	// +kubebuilder:validation:MaxItems=5
-	PrivateIPv4CIDRs []string `json:"privateIPv4CIDRs,omitempty"`
-	// Default size of Subnet based upon estimated workload count.
-	// Defaults to 26.
-	// +kubebuilder:default=26
-	DefaultIPv4SubnetSize int `json:"defaultIPv4SubnetSize,omitempty"`
-	// DefaultSubnetAccessMode defines the access mode of the default SubnetSet for PodVM and VM.
-	// Must be Public or Private.
-	// +kubebuilder:validation:Enum=Public;Private
-	DefaultSubnetAccessMode string `json:"defaultSubnetAccessMode,omitempty"`
+	NsxProject string `json:"nsxProject,omitempty"`
+
+	// VpcConnectivityProfile ID. This profile has configuration related to create VPC transit gateway attachment.
+	VpcConnectivityProfile string `json:"vpcConnectivityProfile,omitempty"`
```

Comment:
Let us use VPC, not Vpc in Golang.

Reply:
Updated, the field name in vsphere-automation-sdk-go is `VpcConnectivityProfile`.